public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
so i remember trying to do what you did before but having some bug and just deciding to go with ng-click because time - do you have the forms disappearing if you click the link again?